core/src/main/scala/filodb.core/reprojector/MapDBMemTable.scala

```diff
@@ -84,6 +84,8 @@ class MapDBMemTable[K](val projection: RichProjection[K], config: Config) extend
 
   def numRows: Int = rowMap.size().toInt
 
+  def forceCommit(): Unit = {}
+
   def clearAllData(): Unit = {
     logger.info(s"MemTable: ERASING ALL TABLES!!")
```

Comment:
I hope this never makes it into Splunk or a production environment.... it's really just for tests  :)